src/identity.cc

```diff
@@ -786,9 +991,9 @@ TRITONBACKEND_ModelInstanceExecute(
         continue;
       }
 
-      // Step 2. Get the output buffer. We request a buffer in CPU
-      // memory but we have to handle any returned type. If we get
-      // back a buffer in GPU memory we just fail the request.
+      // Step 2. Get the output buffer. We request a buffer in CPU memory but we
+      // have to handle any returned type. If we get back a buffer in GPU memory
+      // we just fail the request.
```

Comment:
Is this comment still valid? We can handle output on GPU

Reply:
Fixed